src/test/addrman_tests.cpp

```diff
@@ -15,9 +15,13 @@ class CAddrManTest : public CAddrMan
     uint64_t state;
 
 public:
-    CAddrManTest()
+    CAddrManTest(bool makeDeterministic=true)
```

Comment:
I've added the parameter `bool makeDeterministic=true` in case a future test explicitly doesn't want to call `MakeDeterministic()`.